### PR TITLE
Add missing button to force Trakt sync

### PR DESCRIPTION
**Pull request by fernandog**
@p0psicles your code was there but the button not.

Also i fixed the remove from collection as it wasn't using status like the add to collection

### CODE REVIEW

File: medusa/trakt_checker.py [hunk 1]
```diff
@@ -206,19 +206,21 @@ def sync_library(self):
                 logger.log(u"Synced Trakt collection", logger.DEBUG)
 
     def remove_episode_trakt_collection(self, filter_show=None):
-        """Remove episode from trakt collection."""
+        """Remove episode from trakt collection that no longer has media file (location)."""
```
Comment (p0psicles) [inline medusa/trakt_checker.py]: You don't need to explain the method in detail in your summary. If you really want to explain conditions in your method, then provide that on a separate line, with one white line in between. Or just add a comment above the condition.

File: medusa/trakt_checker.py [hunk 2]
```diff
@@ -249,15 +251,14 @@ def remove_episode_trakt_collection(self, filter_show=None):
                                    (error=e.message), logger.INFO)
 
     def add_episode_trakt_collection(self):
-        """Add all existing episodes to Trakt collections."""
+        """Add all existing episodes to Trakt collections that has media file (location)."""
```
Comment (p0psicles) [inline medusa/trakt_checker.py]: Same here.

File: medusa/trakt_checker.py [hunk 3]
```diff
@@ -396,7 +397,7 @@ def add_episode_watchlist(self):
                                    (error=e.message), logger.INFO)
 
     def add_show_watchlist(self):
-        """Add show to Trakt watchlist."""
+        """Add show to Trakt watchlist if is in show list."""
```
Comment (p0psicles) [inline medusa/trakt_checker.py]: Same here.

File: medusa/trakt_checker.py (hunk 1 above)
Comment (fernandog) [inline medusa/trakt_checker.py]: ok. will change it

File: medusa/trakt_checker.py (hunk 2 above)
Comment (fernandog) [inline medusa/trakt_checker.py]: ok. will change it

File: medusa/trakt_checker.py [hunk 4]
```diff
@@ -206,19 +206,25 @@ def sync_library(self):
                 logger.log(u"Synced Trakt collection", logger.DEBUG)
 
     def remove_episode_trakt_collection(self, filter_show=None):
-        """Remove episode from trakt collection."""
+        """Remove episode from trakt collection.
+
+        For episodes that no longer has media file (location)
```
Comment (p0psicles) [inline medusa/trakt_checker.py]: Have a..

File: medusa/trakt_checker.py [hunk 5]
```diff
@@ -206,19 +206,25 @@ def sync_library(self):
...
+        :param filter_show: optional. Only remove from given shows
```
Comment (p0psicles) [inline medusa/trakt_checker.py]: `Only remove from given shows` this log is not really explaining anything.

File: medusa/trakt_checker.py [hunk 6]
```diff
@@ -249,15 +255,17 @@ def remove_episode_trakt_collection(self, filter_show=None):
                                    (error=e.message), logger.INFO)
 
     def add_episode_trakt_collection(self):
-        """Add all existing episodes to Trakt collections."""
+        """Add all existing episodes to Trakt collections.
+
+        For episodes that has media file (location)
```
Comment (p0psicles) [inline medusa/trakt_checker.py]: have a..

File: medusa/trakt_checker.py [hunk 7]
```diff
@@ -206,19 +206,25 @@ def sync_library(self):
...
                             b'e.season, e.episode, e.status ' \
-                            b'FROM tv_episodes AS e, tv_shows AS s WHERE s.indexer_id = e.showid and e.location = ""'
+                            b'FROM tv_episodes AS e, tv_shows AS s WHERE s.indexer_id = e.showid and e.location = "" ' \
+                            b'AND e.status in ({0})'.format(','.join(selection_status))
             if filter_show:
                 sql_selection += b' AND s.indexer_id = ? AND e.indexer = ?'
                 params = [filter_show.indexerid, filter_show.indexer]
 
-            sql_result = main_db_con.select(sql_selection, params)
+            sql_result = main_db_con.select(sql_selection, Quality.DOWNLOADED + Quality.ARCHIVED + params)
```
Comment (p0psicles) [inline medusa/trakt_checker.py]: did you test this? As I don't think this will work. You can only use list + list here.

Comment (fernandog) [inline medusa/trakt_checker.py]: @p0psicles yes, Quality.DOWNLOADED and Quality.ARCHIVED are lists

Comment (p0psicles) [inline medusa/trakt_checker.py]: Ooh okay, then no prob. Thought they where constants.

**Pull request merged**